splunk-otel-android/src/main/java/com/splunk/rum/SessionId.java

```diff
@@ -55,12 +56,19 @@ String getSessionId() {
             //if this returns false, then another thread updated the value already.
```

Comment:
There's a 
```java
if (currentValue == null || sessionExpired()) {
```
condition a couple of lines above - @jkwatson is it really possible for `currentValue` to be null? The value gets initialized in the constructor, it shouldn't ever return null

Reply:
I agree. That is probably some extra guarding that isn't actually necessary.